docs/sources/setup-grafana/configure-security/planning-iam-strategy/index.md

```diff
@@ -28,6 +28,22 @@ Integrating with an IAM provider provides the following benefits:
 
 In order to plan an integration with Grafana, assess your organization's current needs, requirements, and any existing IAM solutions being used. This includes thinking about how roles and permissions will be mapped to users in Grafana and how users can be grouped to access shared resources.
 
+### Authentication Feature table
```

Comment:
outstanding issue:

should we add explainations to each of the numbers?

Such as 11 - active sync. Or should we just leave it as is without explaining and refer to each column separately by linking to documentation elsewhere

Reply:
Thoughts:
* we already have a similar table underneath [this doc section](https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/), I think we should extend that instead of adding a new table;
* I think we could remove the numbers and link to the corresponding section from each column heading.

Reply:
> I think we could remove the numbers and link to the corresponding section from each column heading.

Gotcha!

Reply:
@IevaVasiljeva ah great. I did not see that table. We should definitely extend that table instead!

Reply:
@eleijonmarck , @IevaVasiljeva - just checking on this PR and seeing if there is anything I can do. I know we're busy this week hacking, just reaching out.

Reply:
@chri2547 hey chris. il be working on this this week. just putting it together somewhat.

Thinking of changing to not include which versions we support as it is hareder to maintain. the table would be from some version back and we can then only add the features we support that version

Reply:
@eleijonmarck , sounds good, thank you!